### ISSUE TITLE
Lost of files to reproduce the tutorial

### ISSUE THREAD

**Issue by jefferyUstc** [rajewsky-lab/openst#33]
Hi, openst team, great work for spatial omics, I'm working on the tutorial provided in the docs (Adult mouse hippocampus), when it comes to the [preprocessing_sequencing](https://rajewsky-lab.github.io/openst/main/examples/adult_mouse/preprocessing_sequencing/) , It seems I can't find the necessary files to complete this step below. Could you please provide the download links for <path_to_rRNA_sequences.fa> and <path_to_phiX_genome.fa>?
```
spacemake config add_species \
   --name mouse \
   --reference genome \
   --sequence <path_to_genome.fa> \
   --annotation <path_to_genome_annotation.gtf>

spacemake config add_species \
   --name mouse \
   --reference rRNA \
   --sequence <path_to_rRNA_sequences.fa>

spacemake config add_species \
   --name mouse \
   --reference phiX \
   --sequence <path_to_phiX_genome.fa>
```

**Comment by danilexn** [comment]
Hi! We are uploading all genome/annotation files and will be available in a few minutes. Thanks for the feedback!

**Comment by danilexn** [comment]
This was fixed by #34
